manager/controllers/system_delete.go

```diff
@@ -28,9 +28,9 @@ func SystemDeleteHandler(c *gin.Context) {
 		return
 	}
 	var systemInventoryID []string
-	tx := database.Db.BeginTx(base.Context, nil)
+	tx := database.Db.WithContext(base.Context).Begin()
 
-	defer tx.RollbackUnlessCommitted()
+	defer tx.Rollback()
```

Comment:
It seems to have different meaning. Will it work correctly?

Reply:
`RollbackUnlessCommitted` was removed, but IMO this should work same, it will rollback all error cases and once transaction is commited, it will not be rollbacked.